src/routes/index.js

```diff
@@ -3,6 +3,7 @@ import { Switch, Route, Redirect } from 'react-router-dom'
 import web3 from 'web3'
 
 import MarketListPage from 'containers/MarketListPage'
+// import MarketListPage from 'routes/marketlist/container/MarketList'
```

Comment:
Once the Layout is implemented properly in the next step we should use this reference.